kroxylicious/src/test/java/io/kroxylicious/proxy/internal/FilterHandlerTest.java

```diff
@@ -607,6 +612,80 @@ void testDropResponse() {
      */
     @Test
     void sendRequest() {
```

Comment:
Are we missing cases for things like null header or body? 

part of me thinks those are trivial and thus not worth testing the other part of me says they are part of the API contract and thus we should have the tests to catch behaviour changes.

Reply:
tests added.